Core/src/main/java/com/plotsquared/core/PlotSquared.java

```diff
@@ -2014,6 +2017,23 @@ public int getPlotCount() {
         return Collections.unmodifiableSet(set);
     }
 
+    /**
+     * Check if the chunk uses vanilla/non-PlotSquared generation
+     *
+     * @param world            World name
+     * @param chunkCoordinates Chunk coordinates
+     * @return True if the chunk uses non-standard generation, false if not
+     */
+    public boolean isNonStandardGeneration(@NotNull final String world,
```

Comment:
does non-standard mean non-plotsquared?

Reply:
No, it means anything that isn't a normal (standard) plot world